app/pages/sections/order-items/additional-services/order-item/flat/patient/ui-tests/general.ui.test.js

```diff
@@ -5,45 +5,48 @@ import content from '../manifest.json';
 import { solutionsApiUrl } from '../../../../../../../../config';
 import { nockAndErrorCheck, setState, authTokenInSession } from '../../../../../../../../test-utils/uiTestHelper';
 import { sessionKeys } from '../../../../../../../../helpers/routes/sessionHelper';
-import AdditionalServicePageModel from '../../additionalServicesPageModel';
```

Comment:
Unfortunately it looks like you've undone @kiran-mastek's recent work to refactor to the page object model pattern. It would be better to use/reinstate @kiran-mastek's changes and update the tests as necessary.

You may need to update the page model.

Reply:
Reinstated Kiran's changes